packages/components/tests/dummy/app/templates/components/tabs.hbs

```diff
@@ -52,14 +54,23 @@
   <dl class="dummy-component-props" aria-labelledby="component-api-tab">
     <dt>count <code>string</code></dt>
     <dd>
-      <p>Displays an optional count in the Tab.</p>
-      <p>Accepts the text value that should go in the badge counter.</p>
+      <p>
+        Displays an optional
+        <a href="/components/badge-count" target="_blank" rel="noopener noreferrer">BadgeCount</a>
+        in the Tab.
+      </p>
+      <p>
+        Accepts the text value that should go in the
+        <a href="/components/badge-count" target="_blank" rel="noopener noreferrer">BadgeCount</a>.
+      </p>
     </dd>
 
     <dt>icon</dt>
     <dd>
-      <p>Displays an optional FlightIcon.</p>
-      <p>Accepts the name of the FlightIcon.</p>
+      <p>Displays an optional
+        <a href="https://flight-hashicorp.vercel.app/" target="_blank" rel="noopener noreferrer">FlightIcon</a>.</p>
+      <p>Accepts the name of the
+        <a href="https://flight-hashicorp.vercel.app/" target="_blank" rel="noopener noreferrer">FlightIcon</a>.</p>
```

Comment:
probably I would avoid the duplication/repetition doing something like this:
```suggestion
      <p>Displays an optional icon in the tab. Accepts the name of the
        <a href="https://flight-hashicorp.vercel.app/">FlightIcon</a>.</p>
```

Reply:
I see you applied the other patch/suggestion but not this one. Intentional or missed it?

Reply:
I missed it sorry

Reply:
I just added it to the main feature branch